src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -190,7 +178,7 @@ private void requestMaterials()
             return;
         }
 
-        final AbstractBuildingWorker buildingWorker = getOwnBuilding();
+	final AbstractBuildingWorker buildingWorker = getOwnBuilding();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)